modules/@ergonode/core/src/store/list/actions.js

```diff
@@ -7,19 +7,68 @@ import {
 } from './mutations';
 
 export default {
+    setDisabledScopeElement: ({
+        state,
+        commit,
+    }, {
+        scope = 'default',
+        disabledElement,
+    }) => {
+        if (typeof state.disabledElements[scope] === 'undefined') {
+            commit(types.SET_DISABLED_ELEMENT_SCOPE, scope);
+        }
+
+        if (typeof state.disabledElements[scope][disabledElement.languageCode] === 'undefined') {
+            commit(types.SET_DISABLED_ELEMENT_SCOPE_LANGUAGE, {
+                scope,
+                languageCode: disabledElement.languageCode,
+            });
+        }
+
+        commit(types.SET_DISABLED_SCOPE_ELEMENT, {
+            scope,
+            ...disabledElement,
+        });
+    },
+    setDisabledScopeElements: ({
+        state,
+        commit,
+    }, {
+        scope = 'default',
+        disabledElements,
+    }) => {
+        commit(types.SET_DISABLED_ELEMENTS, {
+            ...state.disabledElements,
+            [scope]: disabledElements,
+        });
+    },
+    removeDisabledScopeElements: ({
+        commit,
+    }, payload) => {
+        commit(types.REMOVE_DISABLED_SCOPE_ELEMENTS, payload);
+    },
+    /**
+     * @deprecated since 1.1.0
+     */
     setDisabledElement: ({
         commit,
     }, payload) => {
         commit(types.SET_DISABLED_ELEMENT, payload);
     },
+    /**
+     * @deprecated since 1.1.0
+     */
     setDisabledElements: ({
         commit,
     }, disabledElements) => {
         commit(types.SET_DISABLED_ELEMENTS, disabledElements);
     },
+    /**
+     * @deprecated since 1.1.0
+     */
     removeDisabledElement: ({
         commit,
     }, payload) => {
-        commit(types.REMOVE_DISABLED_ELEMENT, payload);
+        commit(types.REMOVE_DISABLED_SCOPE_ELEMENT, payload);
     },
```

Comment:
deprecated ? but you made changes to this method, so the old method should remain unchanged, just like the two methods above.